core/src/main/java/gatt/CoroutinesGatt.kt

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2020 JUUL Labs, Inc.
+ */
+
+@file:Suppress("RedundantUnitReturnType")
```

Comment:
Still needed? I don't see any  explicit `Unit` return types.

Reply:
Good catch. I'll remove it. 👍